### PR TITLE
Document package building, import COPYRIGHT, LICENSE and README files

### LINKED ISSUE #1: Package for CentOS 7
Reference: https://ggus.eu/index.php?mode=ticket_info&ticket_id=136695

The structure of rpm must be compliant with ARGO guidelines:
https://docs.google.com/document/d/1fDqO0LPjRlX68D_jDm3ulxsc0J17XoZqAjMnjRhDfHI/edit

### PR DESCRIPTION
**Pull request by gwarf**
Document package building (working RPM to fix #1, broken DEB).
Import COPYRIGHT, LICENSE and README files

### CODE REVIEW

**Comment by gwarf** [comment]
@lfield does this PR look reasonable to you? (especially the updated COPYRIGHT, LICENSE.txt and README)

**Review by pandreetto — DISMISSED** [review]
For me it's fine.
I'm going to test the build as it is described, in case I'll send some comments

**Comment by gwarf** [comment]
@brucellino can you please review this PR?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+docker run --rm -v $(pwd):/source -it centos:7
+yum install -y rpm-build make rsync gcc openldap-devel
+cd /source && make rpm
+```
+
+The RPM will be available into the `build/RPMS` directory.
+
+### Building a deb
+
+**The alien-based deb pacakge creation is currently broken (rpm task doesn't work on Ubuntu/Debian)**
```
Comment (brucellino) [inline README.md]: It would be good for future us to add the name of the distro on which it currently doesn't work

File: README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# nagios-plugins-bdii
+
+This nagios probe is used to monitor BDII.
+It provides check_bdii_entries support for Nagios.
+
+BDII documentation is available here: https://gridinfo-documentation.readthedocs.io/
+
+## Building packages
```
Comment (brucellino) [inline README.md:8]: mention that they are built and delivered by Travis too and can be found in the releases part of the repo.

**Review by brucellino — CHANGES_REQUESTED** [review]
I suggest some minor changes to the readme to protect future us from now us.

File: README.md (hunk 1 above)
Comment (gwarf) [reply README.md]: It's on all Ubuntu/Debian that it won't work as it is in fact.

File: README.md (hunk 2 above)
Comment (gwarf) [reply README.md:8]: They aren't for the time being in fact, no travis yet for thi repo :)
Please open a ticket to track this.

**Review by brucellino — APPROVED** [review]

**Pull request merged**